MediaBrowser.Providers/TV/TheTVDB/TvdbEpisodeImageProvider.cs

```diff
@@ -50,27 +50,22 @@ public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, Cancell
             var language = item.GetPreferredMetadataLanguage();
             if (series != null && TvdbSeriesProvider.IsValidSeries(series.ProviderIds))
             {
-                var episodeTvdbId = episode.GetProviderId(MetadataProviders.Tvdb);
-
                 // Process images
                 try
                 {
+                    var episodeInfo = new EpisodeInfo
+                    {
+                        IndexNumber = episode.IndexNumber.Value,
+                        ParentIndexNumber = episode.ParentIndexNumber.Value,
+                        SeriesProviderIds = series.ProviderIds
+                    };
+                    string episodeTvdbId = await _tvDbClientManager
+                        .GetEpisodeTvdbId(episodeInfo, language, cancellationToken).ConfigureAwait(false);
                     if (string.IsNullOrEmpty(episodeTvdbId))
                     {
-                        var episodeInfo = new EpisodeInfo
-                        {
-                            IndexNumber = episode.IndexNumber.Value,
-                            ParentIndexNumber = episode.ParentIndexNumber.Value,
-                            SeriesProviderIds = series.ProviderIds
-                        };
-                        episodeTvdbId = await _tvDbClientManager
-                            .GetEpisodeTvdbId(episodeInfo, language, cancellationToken).ConfigureAwait(false);
-                        if (string.IsNullOrEmpty(episodeTvdbId))
-                        {
-                            _logger.LogError("Episode {SeasonNumber}x{EpisodeNumber} not found for series {SeriesTvdbId}",
-                                episodeInfo.ParentIndexNumber, episodeInfo.IndexNumber, series.GetProviderId(MetadataProviders.Tvdb));
-                            return imageResult;
-                        }
+                        _logger.LogError("Episode {SeasonNumber}x{EpisodeNumber} not found for series {SeriesTvdbId}",
```

Comment:
When splitting parameters into multiple lines every parameter should get its own line.

Reply:
Also the first one?

Reply:
Yes, or the analyzers will scream at you.